### PR TITLE
Log level bug fix and 404 exception handling

### PR DESCRIPTION
**Pull request by marco-scoppetta**
- again change to GraqlParse to batchParse
- changes to importController in Engine
- fixed a bug in logs printing DEBUG messaged when set to INFO
- added 404 exception handling in engine
- new version of Java Spark with MIME guessing to return proper Content-type

### CODE REVIEW

File: grakn-dashboard/src/controllers/visualiser.js [hunk 1]
```diff
@@ -229,8 +229,6 @@ export default {
                 else
                     this.selectedProps.push(p);
 
-                console.log("ehi proprieta' "+JSON.stringify(this.selectedProps));
```
Comment (fppt) [inline grakn-dashboard/src/controllers/visualiser.js]: Hahaha I like how this got committed :)

File: grakn-engine/src/main/java/ai/grakn/engine/controller/ImportController.java [hunk 2]
```diff
@@ -211,47 +207,48 @@ private void importDataFromFile(String dataFile, Loader loaderParam) {
...
+    private Object consumeInsertEntity(Iterator<Object> batchIterator, Loader loader) {
+        Object var = null;
+        List<Var> insertQuery = new ArrayList<>();
+        while (batchIterator.hasNext()) {
+            var = batchIterator.next();
+            if (var instanceof Var) {
+                insertQuery.add(((Var) var));
+                processedEntities.incrementAndGet();
+            } else
+                break;
```
Comment (fppt) [inline grakn-engine/src/main/java/ai/grakn/engine/controller/ImportController.java]: Why is there a break here ? What does `batchIterator` contain that is not a `var` ?

File: grakn-engine/src/main/java/ai/grakn/engine/controller/ImportController.java [hunk 3]
```diff
@@ -211,47 +207,48 @@ private void importDataFromFile(String dataFile, Loader loaderParam) {
...
         }
-
-        if (ready) {
-            processedRelations.incrementAndGet();
-            loader.add(Graql.insert(var));
+        List<Var> insertQuery;
+        if (!var.equals(INSERT_KEYWORD))
+            throw new GraknEngineServerException(500, "Match statement not followed by any Insert.");
```
Comment (fppt) [inline grakn-engine/src/main/java/ai/grakn/engine/controller/ImportController.java]: Maybe this error should be in the error class ?

File: grakn-engine/src/main/java/ai/grakn/engine/controller/ImportController.java (hunk 2 above)
Comment (marco-scoppetta) [inline grakn-engine/src/main/java/ai/grakn/engine/controller/ImportController.java]: The new `parseBatchLoad` from queryParser returns a list of Strings("insert" | "match") and Vars, that's why I am using an `Iterator<Object>`. When we encounter a String then we know it's the beginning of a new query, so we stop stacking the Vars belonging to the current query.

File: grakn-engine/src/main/java/ai/grakn/engine/controller/ImportController.java (hunk 3 above)
Comment (marco-scoppetta) [inline grakn-engine/src/main/java/ai/grakn/engine/controller/ImportController.java]: I will move it for the next PR

**Pull request merged**